### PR TITLE
LPS-172475 Add unavailable warnings

### PR DESCRIPTION
**Pull request by sandrodw3**
This is rebased on top of https://github.com/liferay-echo/liferay-portal/pull/10851, only LPS-172475 commits need to be reviewed

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/FormWithControls.js [hunk 1]
```diff
@@ -52,22 +43,35 @@ const FormWithControls = React.forwardRef(({children, item, ...rest}, ref) => {
...
+	}
+
+	if (Liferay.FeatureFlags['LPS-169923'] && !formHasPermissions(item)) {
+		return <PermissionRestrictionMessage />;
+	}
+
+	const isMapped = formIsMapped(item);
+
+	if (isEmpty || !isMapped) {
+		return <FormEmptyState isMapped={isMapped} item={item} />;
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/FormWithControls.js:88]: 👏

**Comment by victorg1991** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/FormWithControls.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/layout-data-items/FormWithControls.js:88]: 🥰

**Pull request closed without merging**